runtime/altair/src/lib.rs

```diff
@@ -1497,11 +1497,8 @@ pub type Executive = frame_executive::Executive<
 	Runtime,
 	AllPalletsWithSystem,
 	(
-		pallet_multisig::migrations::v1::MigrateToV1<Runtime>,
-		pallet_preimage::migration::v1::Migration<Runtime>,
-		pallet_democracy::migrations::v1::Migration<Runtime>,
-		pallet_scheduler::migration::v3::MigrateToV4<Runtime>,
 		pallet_interest_accrual::migrations::v2::Migration<Runtime>,
+		pallet_loans::migrations::Migration<Runtime>,
```

Comment:
I've removed the other migrations because they were failing in the `pre_upgrade()` phase (checked one by one), which I understand is because they were already applied.

**PLEASE, DOUBLE-CHECK THIS**

Reply:
pallet-interets-accrual should be removed to

Reply:
comment below

Reply:
You are right, I guess. https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Faltair.api.onfinality.io%2Fpublic-ws#/chainstate

Reply:
You can see the state of altair.

Reply:
But this is weird. Why where all the others still included? Are they mentioned in the last release notes of Altair
?

Reply:
I think those were included in the last release but not removed after it. If I leave them there, they complain that they were already migrated. I suppose we added the interest accrual to that list without cleaning the previous migrations.

Reply:
What is weirdest is that the `interest-accrual` pallet seems empty with version 0. Is it ok? 🤔

Reply:
Regarding the versioning of this migration. This migration tries to remove the whole pallet, in order to remove it from the codebase, and use the new one. In that case, makes sense to version it? I mean, If I version this, I would store some version number on the pallet storage that I don't want, because I'll like to start from the scratch with the new pallet

Reply:
(resolution about my concerns in the bottom)

Reply:
I think storage versioning is a separate concern from what we're doing here, which is purely a purge of old data.